packages/server/graphql/intranetSchema/mutations/enableSAMLForDomain.ts

```diff
@@ -0,0 +1,36 @@
+import {GraphQLNonNull, GraphQLString, GraphQLBoolean} from 'graphql'
+import getRethink from '../../../database/rethinkDriver'
+
+const enableSAMLForDomain = {
+  type: new GraphQLNonNull(GraphQLBoolean),
+  description: 'Enable SAML for domain',
+  args: {
+    url: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    domain: {
+      type: GraphQLNonNull(GraphQLString),
+    },
+    metadata: {
+      type: GraphQLNonNull(GraphQLString),
+    },
+    cert: {
+      type: GraphQLNonNull(GraphQLString),
+    }
+  },
+  async resolve(_source, {url, domain, metadata, cert}) {
+    const r = await getRethink()
+    const normalizedDomain = domain.toLowerCase()
+    await r
+      .table('SAML')
+      .insert({
+        domain: normalizedDomain,
+        url: url,
+        metadata: metadata,
+        cert: cert
+      })
+      .run()
```

Comment:
@NikAiyer this will error with `"Cannot return null for non-nullable field Mutation.enableSAMLForDomain."`. The resolver returns null even as the mutation is defined to return a boolean. Please update to `return true`

Reply:
Got it. Will add now.